src/main/scala/scalismo/registration/LandmarkRegistration.scala

```diff
@@ -97,16 +97,93 @@ object LandmarkRegistration {
    *
    * @param landmarks sequence of corresponding landmarks
    */
+  def rigid3DLandmarkRegistration(landmarks: Seq[(Point[_3D], Point[_3D])]): RigidTransformation[_3D] = rigid3DLandmarkRegistration(landmarks, origin3D)
```

Comment:
Center is defaulted here to origin. Does not match with the strategy to explicitly state the center everywhere.